### PR TITLE
Those value are incorrect.

### PR DESCRIPTION
**Pull request by taka-oi**

### CODE REVIEW

File: WindowsServerDocs/networking/windows-time-service/windows-server-2016-improvements.md [hunk 1]
```diff
@@ -54,11 +54,11 @@ The following describes the changes in default configuration between Windows 201
 |**Standalone Client**||||
 | |*Time Server*|NA|time.windows.com|time.windows.com|
 | |*Polling Frequency*|NA|Once a day|Once a week|
-| |*Clock Update Frequency*|NA|Once a day|Once a week|
+| |*Clock Update Frequency*|NA|Once a day|Once a hour|
```
Comment (illfated) [inline WindowsServerDocs/networking/windows-time-service/windows-server-2016-improvements.md]: I have not checked the validity of these values, but the grammar needs adaption:
```suggestion
| |*Clock Update Frequency*|NA|Once a day|Once an hour|
```
("an" is used before words starting with a vowel _when pronounced_ in English. The H in hour is silent, so it is pronounced like "our".)

File: WindowsServerDocs/networking/windows-time-service/windows-server-2016-improvements.md [hunk 2]
```diff
@@ -54,11 +54,11 @@ The following describes the changes in default configuration between Windows 201
 |**Standalone Client**||||
 | |*Time Server*|NA|time.windows.com|time.windows.com|
 | |*Polling Frequency*|NA|Once a day|Once a week|
-| |*Clock Update Frequency*|NA|Once a day|Once a week|
+| |*Clock Update Frequency*|NA|Once a day|Once a hour|
 |**Domain Controller**||||
 | |*Time Server*|PDC/GTIMESERV|NA|PDC/GTIMESERV|
 | |*Polling Frequency*|64 -1024 seconds|NA|1024 - 32768 seconds|
-| |*Clock Update Frequency*|Once a day|NA|Once a week|
+| |*Clock Update Frequency*|Once a second|NA|Once a hour|
```
Comment (illfated) [inline WindowsServerDocs/networking/windows-time-service/windows-server-2016-improvements.md]: Same here:
```suggestion
| |*Clock Update Frequency*|Once a second|NA|Once an hour|
```

**Comment by eross-msft** [comment]
@dcuomo  Please review these changes and let me know if they are okay to merge. Thanks!

**Comment by illfated** [comment]
@taka-oi : May I ask you to commit the suggested changes?

**Comment by illfated** [comment]
Just as a side note, the entire page could do with some proofreading, but that is way outside the scope of this PR.

**Comment by illfated** [comment]
#sign-off

**Comment by eross-msft** [comment]
Sending this question to PM. Will respond after I get response.

**Comment by PRMerger15** [comment]
@eross-msft : Thanks for your contribution! The author(s) have been notified to review your proposed change. @dahavey

**Pull request merged**